plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,516 @@
+	NUMCOL=5
+	LINES=$(wc -l <<< "$member_elements")
+	while IFS=$'\n' read -ra graphs ; do
+		element+=( $graphs )
+	done <<< "$member_elements"
+
+	echo "multigraph vnx_emc_v3_calls_s"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+1]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v3_usec_call"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+3]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v3_op_percent"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+4]}"
+	done
+
+	element=()
+
+#nfs.v4.op data
+	member_elements=$($SSH server_stats $server -monitor nfs.v4.op -count 1 -terminationsummary no -titles never | sed -ne 's/^.*v4/v4/p')
+	NUMCOL=5
+	LINES=$(wc -l <<< "$member_elements")
+	while IFS=$'\n' read -ra graphs ; do
+		element+=( $graphs )
+	done <<< "$member_elements"
+
+	echo -e "\nmultigraph vnx_emc_v4_calls_s"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+1]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v4_usec_call"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+3]}"
+	done
+
+	echo -e "\nmultigraph vnx_emc_v4_op_percent"
+	for ((i=0; i<$((LINES)); i++ )); do
+		echo "${server}_${element[i*$NUMCOL]}".value "${element[i*$NUMCOL+4]}"
+	done
+
+	element=()
+
+#nfs.client data
+	echo -e "\nmultigraph vnx_emc_nfs_client_ops_s"
+        member_elements=$($SSH server_stats server_2 -monitor nfs.client -count 1 -terminationsummary no -titles never | sed -ne 's/^.*id=//p')
+	NUMCOL=9
+        LINES=$(wc -l <<< "$member_elements")
+	while IFS=$'\n' read -ra graphs; do
+		element+=($graphs)
+	done  <<< "$member_elements"
+	for (( i=0; i<$((LINES)); i++ )); do
+		client=$( echo ${element[i*$NUMCOL]} | sed -ne 's/\./_/pg')
+               echo "${server}_${client}_r".value "${element[$i*$NUMCOL+2]}"
+               echo "${server}_${client}_w".value "${element[$i*$NUMCOL+3]}"
+               echo "${server}_${client}_t".value "${element[$i*$NUMCOL+1]}"
+               echo "${server}_${client}_s".value "${element[$i*$NUMCOL+4]}"
+        done
+	echo -e "\nmultigraph vnx_emc_nfs_client_kib_s"
+	for (( i=0; i<$((LINES)); i++ )); do
+		client=$( echo ${element[i*$NUMCOL]} | sed -ne 's/\./_/pg')
+               echo "${server}_${client}_r".value "${element[$i*$NUMCOL+6]}"
+               echo "${server}_${client}_w".value "${element[$i*$NUMCOL+7]}"
+               echo "${server}_${client}_t".value "${element[$i*$NUMCOL+5]}"
+        done
+	echo -e "\nmultigraph vnx_emc_nfs_client_avg_usec"
+	for (( i=0; i<$((LINES)); i++ )); do
+		client=$( echo ${element[i*$NUMCOL]} | sed -ne 's/\./_/pg')
+		echo "${server}_${client}".value "${element[$i*$NUMCOL+8]}"
+	done
+	fi
+done
+if [ -z $nas_server_ok ]; then
+	echo "No active data movers!"
+	exit 1
```

Comment:
Sorry for one more stupid question, but how do i signal about error without "exit 1"?